bin/add_callers_to_scout_custom.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+
+import argparse
+from collections import defaultdict
+
+
+def parse_arguments():
+    """Parse command-line arguments."""
+    parser = argparse.ArgumentParser(
+        description="Copy caller names from SVDB INFO.set to INFO.SCOUT_CUSTOM in SVDB merged VCFs"
+    )
+    parser.add_argument(
+        "--merged_vcf", required=True, help="Path to the merged VCF file."
+    )
+    parser.add_argument(
+        "--callers", required=True, help="Comma-separated list of used callers."
+    )
+    return parser.parse_args()
+
+
+def process_vcf(merged_vcf, used_callers):
+    """Process the VCF file and copy caller information to SCOUT_CUSTOM variant by variant."""
+    used_callers_set = set(used_callers.split(","))
+
+    with open(merged_vcf, "r") as vcf_file:
+        for line in vcf_file:
+            if line.startswith("#"):
+                print(line.strip())
+                continue
+
+            columns = line.strip().split("\t")
+            chrom, pos, id_, ref, alt, qual, filter_, info, fmt = columns[:9]
+            samples = columns[9:]
+
+            info_dict = parse_info_field(info)
+
+            callers_flag = defaultdict(
+                bool, {"manta": False, "cnvkit": False, "gatk": False}
+            )
+
+            # Parse caller information from the "set" field in INFO
+            set_field = info_dict.get("set", "")
+            callers_in_set = set(set_field.split("-"))
+            for caller in callers_in_set:
+                if caller in callers_flag:
```

Comment:
warning is enough for now. in case the script ends up being run at some other location. we do already end up having SV records without the SVDB tags in some spots